### ISSUE TITLE
failed to create locked file when git clone

### ISSUE THREAD

**Issue by ylc395** [petersalomonsen/wasm-git#23]
version：0.0.4

I'm trying to use wasm-git in a worker, which is spawned in electron renderer.I mouted file system with NODEFS, and call `clone` command . However, I got the following error:

![图片](https://user-images.githubusercontent.com/12264626/113719402-288cdd00-9720-11eb-9fc7-20bed46788e2.png)

Here is the code in the worker script:
```js
FS.mkdir('/working');
FS.mount(NODEFS, { root: '/User/xxxx/.config/my-app' }, '/working');
FS.chdir('/working');
lg.callMain([
    'clone',
    `https://github.com/petersalomonsen/wasm-git.git`,
    'workingTree',
]);
```
Did I miss something? In fact, `.git` is created in physical file system, but not the files in working tree.

<img width="740" alt="图片" src="https://user-images.githubusercontent.com/12264626/113720105-e1ebb280-9720-11eb-849b-5086181d37a8.png">

**Comment by petersalomonsen** [comment]
seems like a bug has been introduced here. just tried with version 0.0.2 here: https://unpkg.com/browse/wasm-git@0.0.2/ where it works for me, but with 0.0.4 it fails. Will try to figure out what has happened here. thanks for reporting :)

**Comment by petersalomonsen** [comment]
Actually it seems that 0.0.4 was built without the last two lines (replacement from 0444 to 0644) in setup.sh.

Would you try the version in the attached file?

[wasmgit.zip](https://github.com/petersalomonsen/wasm-git/files/6266794/wasmgit.zip)

**Comment by ylc395** [comment]
Wow, it works now 👍. Maybe we can realease it as 0.0.5?

**Comment by petersalomonsen** [comment]
released 0.0.5, should be ok now :)

**Comment by petersalomonsen** [comment]
actually use 0.0.6.. 0.0.5 was built in debug mode and is not optimized for size.
